src/MICore/Debugger.cs

```diff
@@ -49,6 +49,8 @@ public class Debugger : ITransportCallback
 
         public bool IsCygwin { get; protected set; }
 
+        public bool IsLocalGdbOnWindows { get; protected set; }
```

Comment:
There are some cases where we can be using a local GDB on Windows where this will not be true. I am guessing this flag matches the cases where you want to send the extra new line, but doesn't quite match when we are using local GDB. So I would suggest changing the name of this flag to 'SendExtraNewLine'.